tests/detect-lua/detect-lua-04/test.yaml

```diff
@@ -0,0 +1,17 @@
+args:
```

Comment:
It seems to me that the pcaps for this test and for `detect-lua-01` are the same. In this case, here, you could add:
```yaml
pcap: ../detect-lua-01/lua-http.pcap
```

Then you can delete the pcap for this test. That would help us reuse the existing pcap and remove duplicated files ;)